todo_app/data/session_items.py

```diff
@@ -13,7 +20,10 @@ def get_items():
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS)
+    All_items = session.get('items', _DEFAULT_ITEMS)
```

Comment:
The variable naming convention for Python is all lowercase and snake_case. So `all_items`